http/handler.go

```diff
@@ -58,7 +58,7 @@ func Handler(core *vault.Core) http.Handler {
 	mux.Handle("/v1/sys/wrapping/unwrap", handleRequestForwarding(core, handleLogical(core, false, wrappingVerificationFunc)))
 	mux.Handle("/v1/sys/capabilities-self", handleRequestForwarding(core, handleLogical(core, true, nil)))
 	mux.Handle("/v1/sys/", handleRequestForwarding(core, handleLogical(core, true, nil)))
-	mux.Handle("/v1/", handleRequestForwarding(core, handleLogical(core, false, nil)))
+	mux.Handle("/v1/", handleCORS(core, handleRequestForwarding(core, handleLogical(core, false, nil))))
```

Comment:
My guess is that we'd want CORS for all calls, right? If so, probably the right thing to do would be to wrap handler generally (like happens with handleHelpHandler).